src/locales/en/chapter2.js

```diff
@@ -94,8 +94,8 @@ export const princess = {
     "Leave us.", // loretta
     "Mary, who are they?", // loretta
     "Report them to your father.", // loretta
-    "You know I can't do that.", // mary
-    "I can't enter the castle from down here.", // mary
+    "No need to do that.", // mary
+    "Even thought they can enter here, they never can enter the castle.", // mary
```

Comment:
OK that makes sense.  

98: "Even though they're here, they can never enter the castle."

It might also be a good place to mention Orthrus for the first time:  "Even though they've found this passage, they'll never get past Orthrus."

Reply:
That's a great idea!
I've added about that to Japanese version too.